app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -32,3 +32,7 @@ a.button--gray, button.button--gray, .button--gray:visited {
 .mb-3 {
   margin-bottom: 3rem;
 }
+
+.margin-or {
```

Comment:
margin or?

Reply:
Its used to create space around the content "Or".

Reply:
Can we name it similar to the others in here e.g. `mt-5`